client/lib/nav-utils/index.js

```diff
@@ -6,6 +6,11 @@ import history from 'lib/history';
 import { parse, stringify } from 'qs';
 import { isEmpty, pick, uniq } from 'lodash';
 
+/**
+ * Internal dependencies
+ */
+import * as navUtils from './index';
```

Comment:
Below we are using `getQuery` directly. I didn't know you could do this. I think its easier to see if used directly (relies on hoisting).

Reply:
Oh is this for `jest.mock` to work. Maybe add a comment?

Reply:
Yes, exactly, this seems to be the only way to mock functions from the same module.

Added a comment in 6dd3c7f9c452cd29f7815020bb3be2bdb2f23d7f.